tests/unit/DeserializerFactoryTest.php

```diff
@@ -15,7 +15,14 @@
 class DeserializerFactoryTest extends \PHPUnit_Framework_TestCase {
 
 	private function buildDeserializerFactory() {
-		return new DeserializerFactory( new DataValueDeserializer(), new BasicEntityIdParser() );
+		return new DeserializerFactory( [
+			'item' => function ( DeserializerFactory $factory ) {
+				return $factory->newItemDeserializer();
+			},
+			'property' => function ( DeserializerFactory $factory ) {
+				return $factory->newPropertyDeserializer();
+			},
```

Comment:
Only `testNewEntityDeserializer` depends on key values `item` and `property`, other tests don't care.

I guess this setup should be moved to `testNewEntityDeserializer` and this method might use `[]` instead.